### PR TITLE
feat(content): Gefüllte Taubenbrust Person Ship

### PR DESCRIPTION
**Pull request by Saugia**
**Content**

## Summary
"And then the mighty said, may he be ultimately cast among us for us to revel and wish upon".
-Someone, probably.

This PR adds Gefu's long-deserved person ship to the game. His person ship will be a Navy Battleship, planned for the future Republic Campaign, similar to how Zitchas flies the future Remnant Heron. All assets are done by him and attributed appropriately. Quotes were selected based on his most common quotes, as well as others found from his history in Discord, and are approved by him. If we want to add more, by all means, suggest! Thank you for your hard work, Gef, you've come a long way, and have made the game look a lot nicer. 

Battleship
Note: Using the Author government, the swizzle will be the normal Red/Orange/Yellow that the Republic uses, which is fitting for it being a Navy ship, and is okay with Gefu.
## Save File
N/A

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/95
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/316

### CODE REVIEW

File: data/persons.txt [hunk 1]
```diff
@@ -1475,3 +1475,223 @@ ship "Modified Boxwing"
...
+	turret -75 108 "Heavy Laser Turret"
+	turret 75 108 "Heavy Laser Turret"
+	turret 0 10 "Wanderer Anti-Missile"
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description `The largest ship made by the navy so far, it is hoped to help repel any future invasions by aliens in human space. Its design leans towards turrets more than guns and has the equivalnt armament of a small fleet.`
```
Comment (roadrunner56) [inline data/persons.txt:1576]:
```suggestion
	description `The largest ship made by the Navy so far, it is hoped to help repel any future invasions by aliens in human space. Its design leans towards turrets more than guns and has the equivalent armament of a small fleet.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by GefullteTaubenbrust2 — APPROVED** [review]

**Comment by Hurleveur** [comment]
Could you mb provide a screenshot of the ship in action?

File: data/persons.txt [hunk 2]
```diff
@@ -1475,3 +1475,223 @@ ship "Modified Boxwing"
...
+	turret -75 108 "Heavy Laser Turret"
+	turret 75 108 "Heavy Laser Turret"
+	turret 0 10 "Wanderer Anti-Missile"
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description `Never underestimate how far an individual is willing to go to modify their favorite spacecraft- sometimes before it even exists.`
```
Comment (roadrunner56) [inline data/persons.txt:1576]:
```suggestion
	description `Never underestimate how far an individual is willing to go to modify their favorite spacecraft - sometimes before it even exists.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Review by Hurleveur — APPROVED** [review]
Looks terrific!

**Pull request merged**